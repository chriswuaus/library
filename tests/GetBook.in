book 111111 long
book 111111
add book test.csv 111111
book 100000 long
book 100000 
book 111111 long
book 111111 
exit
