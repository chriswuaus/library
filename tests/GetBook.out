user: No books in system.

user: No books in system.

user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: No such book in system.

user: No such book in system.

user: 111111: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

user: To Kill a Mockingbird (Harper Lee)

user: Ending Library process.
